notebooks/jax-vit-mixer/jax-vit-mixer.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
<p>why models if you compile only one model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>